docs/guides/advanced/autocomplete.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Autocomplete
+---
+
+# Autocomplete parameter options
+
+If you have a parameter that accepts a defined set of values it's usually best to provide those options using autocomplete. These options are presented to the user in the Coda UI and they can simply click on one instead of entering the value manually.
+
+[View Sample Code][samples]{ .md-button }
+
+
+## Using autocomplete
+
+In the formula editor parameter options show up in a the same pane used for other autocomplete. In the actions builder and sync table settings options are presented in a drop down.
+
+=== "In the formula editor"
+    <img src="../../../images/autocomplete_formula.png" srcset="../../../images/autocomplete_formula_2x.png 2x" class="screenshot" alt="Autocomplete in the formula editor">
+=== "In the action builder"
+    <img src="../../../images/autocomplete_action.png" srcset="../../../images/autocomplete_action_2x.png 2x" class="screenshot" alt="Autocomplete in the action builder">
+=== "In the sync table settings"
+    <img src="../../../images/autocomplete_sync.png" srcset="../../../images/autocomplete_sync_2x.png 2x" class="screenshot" alt="Autocomplete in the sync table settings">
+
+
+
+## Simple options
+
+The simplest way to set this up is to set the `autocomplete` property of the parameter to an array of valid options.
+
+```ts
+coda.makeParameter({
+  type: coda.ParameterType.String,
+  name: "animal",
+  description: "The selected animal.",
+  autocomplete: ["cow", "pig", "sheep"]
+})
+```
+
+This option is only available for `String` parameters.
```

Comment:
Does it not work for Number parameters? Trying to think why it wouldn't.

Reply:
The definition only allows for an array of strings or SimpleAutocompleteOptions:

https://github.com/coda/packs-sdk/blob/a66775f9c084b750277b277e0cb2cee34332499b/api.ts#L254

It would make sense to support number arrays as well, and I can file an FR if you'd like.

Reply:
Ah how silly. This should fix it: https://github.com/coda/packs-sdk/pull/1517

Reply:
Thanks! I've preemptively updated the docs to mention support for numbers.